.github/workflows/test.yml

```
@@ -91,8 +91,16 @@ jobs:
             cli/**
             packages/**
             shim/**
+            Cargo.*
+            .cargo/**
             !**.md
             !**.mdx
+      - name: Test changes
+        id: tests
+        uses: technote-space/get-diff-action@v6
+        with:
+          PATTERNS: |
+            cli/integration_tests/**/*.t
```

```suggestion
            cli/integration_tests/**
```

There are some `*.sh` files or `turbo.json`s in these tests that should probably also trigger a test run